### PR TITLE
Support for location points

### PR DESCRIPTION
**Pull request by jasonmule**
[Review on Reviewable](https://reviewable.io/reviews/ushahidi/platform/911)

### CODE REVIEW

File: application/classes/Ushahidi/FileReader/CSV.php [hunk 1]
```diff
@@ -38,6 +38,7 @@ public function setReaderFactory(ReaderFactory $reader_factory)
 	public function process($file)
 	{
 		$reader = $this->reader_factory->createReader($file);
+		$reader->setEnclosure('\'');
```
Comment (rjmackay) [inline application/classes/Ushahidi/FileReader/CSV.php]: Does this mean we ONLY support single quoted CSVs??

**Comment by rjmackay** [comment]
@willdoran @jasonmule is this documented somewhere in the UI ? ie. locations can be in JSON format in the CSV...

I'm not 100% happy with this since its a weird way to have locations in a CSV.. we should look at handling simple lat,lon and lon,lat formats later.

**Comment by willdoran** [comment]
On the data-import page, there is a description of the format with an example but it might be off-putting/complex. I'm not sure if I described it clearly enough.

**Comment by rjmackay** [comment]
Alright. That'll do for release at least.. is it possible to import lat and lon from separate columns at the moment?

**Comment by aoduor** [comment]
Perhaps this is the source of my failed data uploads. Are the lat/lon values supposed to be in separate columns? or the same column? And what should the column header be?

**Pull request merged**